system-variables.md

```diff
@@ -506,6 +506,16 @@ For more possible values of this variable, see [Authentication plugin status](/s
 
 </CustomContent>
 
+### div_precision_increment <span class="version-mark">New in v8.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
+- Type: Integer
+- Default value: `4`
+- Range: `[0, 30]`
+- This variable is used to control the number of decimal places in the result of a division operation performed using the operator `/`.
```

Comment:
How about use the exactly describe as in mysql's doc:
```
This variable indicates the number of digits by which to increase the scale of the result of division operations performed with the `/` operator.
```

Reply:
Sounds good, I'll change it.